src/components/Sections/AccordionSection/__tests__/AccordionSection.spec.ts

```diff
@@ -0,0 +1,174 @@
+      const lightProps = {
+        dark: false,
+        items: [
+          {
+            title: "test",
+            text: "This is a test text to test text rendering",
+          },
+        ],
+      };
+
+      const { container } = render(AccordionSection, {
+        props: lightProps,
+      });
+      const lightAccordionClassList = container.querySelector(".Accordion")
+        ?.firstElementChild?.classList;
+
+      expect(lightAccordionClassList).toContain("Accordion--Light");
+    });
+    it("renders a div with its class list containing Accordion--Light when the dark property is not set", () => {
+      const darkNotSetProps = {
+        items: [
+          {
+            title: "test",
+            text: "This is a test text to test text rendering",
+          },
+        ],
+      };
+      const { container } = render(AccordionSection, {
+        props: darkNotSetProps,
+      });
+      const darkNotSetClassList = container.querySelector(".Accordion")
+        ?.firstElementChild?.classList;
+      expect(darkNotSetClassList).toContain("Accordion--Light");
+    });
+  });
+  describe("title prop", () => {
+    let container, accordionSection: HTMLElement;
+    beforeAll(() => {
+      container = render(AccordionSection, {
+        props: {
+          title: "This is a title",
+          items: [
+            {
+              title: "I love Luigi",
+              text: "Mario sux",
+            },
+          ],
+        },
+      }).container;
+      accordionSection = container.querySelector(
+        ".Accordion-Section",
+      )! as HTMLElement;
+    });
+    it("when the title prop is set the accordion-section has 3 children", () => {
+      expect(accordionSection?.childElementCount).toBe(3);
+    });
+    it("renders a title and a separator when the title prop is set", () => {
+      expect(accordionSection?.children[0].innerHTML).toBe("This is a title");
+      expect(accordionSection?.children[1].classList).toContain(
+        "Accordion-Section--Separator",
+      );
+    });
+  });
+  describe("items prop", () => {
+    const items = [
+      {
+        title: "one",
+        text: "This is a test text to test text rendering",
+      },
+      {
+        title: "two",
+        text: "This is a test text to test text rendering",
+      },
+    ];
+    let container, firstAccordion: HTMLElement;
+    let accordions: HTMLCollectionOf<Element>;
+    beforeEach(() => {
+      container = render(AccordionSection, {
+        props: { items },
+      }).container;
+      accordions = container.getElementsByClassName("Accordion");
+      firstAccordion = accordions[0] as HTMLElement;
+    });
+
+    it("renders one div with class 'Accordion' for each item in the array", () => {
+      expect(accordions?.length).toEqual(items.length);
+    });
+    it("renders the title of an item in a div with class Accordion--Header", () => {
```

Comment:
You use very specific content and element queries. If they break too often in the future this should be refactored to not so specific queries.